ikarus-webapp/src/features/combatLog/client/combatReport.js

```diff
@@ -0,0 +1,55 @@
+Template.combat_report.helpers({
+  hasCombatLog: function() {
+    return this instanceof CombatLog;
+  },
+
+  getEventHeader: function (combatLog) {
+    var time = moment(this.time).format("H:mm:ss");
+    return time + " " + this.header;
+  },
+
+  parseXml: function (text) {
```

Comment:
This helper could have a name that would be more abstraction level appropriate. getEventContent?

Also not sure if the text content needs to passed to the helper, does "this" point to item being iterated over?